docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to use `earthdata-varinfo` to publish UMM-Var entries to CMR"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Version 2.0.0"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview:"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This version of `earthdata-varinfo` adds functionality to publish records to CMR, along with a single overarching function that wraps the search, download and publication functionality into a single function for the convenience of the end-user. Additionally, the function signatures for `cmr_search.get_granules` and `cmr_search.download_granule` have been updated to accept the full Authorization header, instead of a bearer token, so that they are also compatible with LaunchPad tokens."
```

Comment:
This text look familiar!

I think it's probably worth reframing this introduction to the notebook more for telling an end-user what they will learn by working through it, rather than the release notes for v2.0.0. (Although, it's definitely a great call to mention that the version of `earthdata-varinfo` being used needs to be at least 2.0.0 to run through the notebook)

Reply:
* [ ] Reframe intro to tell end-user about what they will learn from this notebook (leave information about the version so users no what version this applies too)